package.json

```diff
@@ -13,8 +13,9 @@
     "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs"
   },
   "dependencies": {
-    "vue": "^1.0.21",
-    "babel-runtime": "^6.0.0"
+    "babel-runtime": "^6.0.0",
+    "mathjs": "^3.3.0",
```

Comment:
This should be a `devDependency`, as a production build will consolidate all of those into one file. No need for this to be a primary dependency.